### ISSUE TITLE
why the disp_gradient_loss, lr_loss is zero and the total loss is not getting converging?

### ISSUE THREAD

**Issue by zouzhekang** [ClubAI/MonoDepth-PyTorch#27]
I transform this code into tensorflow 2.0 version and trained a driving dataset, then I try your original code  but this question still exist. can you help me? pleeeeeeeeeeeease........
